omod/src/main/webapp/pages/results.gsp

```diff
@@ -156,7 +217,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
         patients.push("${prettify(ui.encodeHtmlContent(it.name))}");
     </script>
     <tr>
-        <td style="display: none">rule_${it.rule.ruleId}</td>
+        <td class="hidden">rule_${it.rule.ruleId}</td>
+        <td class="hidden">${it.patient.patientIdentifier}</td>
```

Comment:
@shivtej1505 What I am saying is that you do not need 2 columns for patient identifier and program ID - it should be the same column identifier and show the relevant value